spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -56,7 +56,7 @@ def get_parser():
              "  - viewer: manual selection a few points followed by interpolation\n"
              "  - fitseg: fit a regularized centerline on an already-existing cord segmentation. It will "
              "interpolate if slices are missing and extrapolate beyond the segmentation boundaries (i.e., every "
-             "axial slice will exhibit a centerline pixel)."
+             "axial slice will exhibit a centerline pixel) if -extrapolation 1 is specified."
```

Comment:
I have one slight wording change:

```python
            " - fitseg: fit a regularized centerline on an already-existing cord segmentation. This method "
            "will interpolate if any slices are missing. Also, if  '-extrapolation 1` is specified, this method will "
            "extrapolate beyond the segmentation boundaries (i.e., every axial slice will exhibit a centerline pixel)."
```

Since this is a long sentence, it might be easy to miss the `-extrapolation 1` part, because it is at the end of the sentence. (People might get lazy and stop reading, hehe.)

So, my suggestion is to split the sentence into two, then add the `-extrapolate 1` part at the start of the sentence.

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4079/commits/6959b7bfcf4c8419cc7758c214dbe1208e81f5e3